### DISCUSSION TITLE
php bin/console debug:router

### DISCUSSION THREAD

**Discussion by On5-Repos** [symfony/symfony#47625] [Q&A]
Hi Guys

I am not sure what has gone wrong but today we did a patch release in DEV and suddenly all routes went missing. 

[root@dev-current]# php bin/console debug:router
 ------ -------- -------- ------ ------
  Name   Method   Scheme   Host   Path
 ------ -------- -------- ------ ------

[root@dev-current]#

I reverted the deployment did everything just no routes, its only on remote DEV server (CENTOS).

On local machine I can run the app without a issue.

Any pointers?

**Comment by MatTheCat** [comment]
First thing that comes to mind is that you could run your application with debug mode disabled without the cache warmed up?
